src/test/java/com/puppycrawl/tools/checkstyle/xpath/XpathMapperTest.java

```diff
@@ -419,7 +419,7 @@ public void testQueryRootNotImplementedAxis() throws Exception {
 
     @Test
     public void testQueryElementNotImplementedAxis() throws Exception {
-        final String xpath = "/CLASS_DEF//following-sibling::METHOD_DEF";
+        final String xpath = "/CLASS_DEF//namespace::*";
```

Comment:
should we mention in doc that https://www.w3schools.com/xml/xpath_axes.asp "namespace" axis will result in exception (not supported as there is no namespaces) ? Or it is kind of obvious ?

Reply:
we can mention all supported axes in `config_filters`

Reply:
it would be awesome, please do in any form you like it to be (link to all possible will be good to have). If some not supported, it would be good to put brief details on why (some reason, or link to issue tracker if it is our todo list).

Reply:
just added list of supported axes, only unsupported is `namespace` axis